src/styles/blocks/card.scss

```diff
@@ -0,0 +1,136 @@
+/* @import 'src/images/star.svg';
+@import 'src/images/star-active.svg'; */
+
+.main__container-cards {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 50px;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--white);
+  border: 1px solid var(--grey);
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  margin-bottom: 46px;
+  margin-right: 48px;
```

Comment:
It's a good practice to use Box Model for styling css, you can read more about it in here: https://www.w3schools.com/css/css_boxmodel.asp